### PR TITLE
Create RESTEasyCLI.yml

### PR DESCRIPTION
**Pull request by suryasr007**

### CODE REVIEW

File: _data/projects/RESTEasyCLI.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/rapidstack/RESTEasyCLI
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- cli
+- yml
+- REST
+upforgrabs:
+  name: help needed
+  link: https://github.com/rapidstack/RESTEasyCLI/issues
```
Comment (shiftkey) [inline _data/projects/RESTEasyCLI.yml]: This link should be to the `help needed` issues: `https://github.com/rapidstack/RESTEasyCLI/labels/help%20wanted`

Comment (suryasr007) [reply _data/projects/RESTEasyCLI.yml]: Oh yeah, I forgot. I will make changes. Thanks for the info.

**Comment by suryasr007** [comment]
Hi @shiftkey 

Is there any way to provide multiple labels?

**Comment by shiftkey** [comment]
> Is there any way to provide multiple labels?

@suryasr007 not currently, see #344 for that discussion

File: _data/projects/RESTEasyCLI.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+desc: Handy REST API client on your terminal
+site: https://github.com/rapidstack/RESTEasyCLI
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- cli
+- yml
+- REST
+upforgrabs:
+  name: help needed
```
Comment (ritwik12) [inline _data/projects/RESTEasyCLI.yml:11]: @suryasr007 Please change this to "help wanted" as your label link is for "help wanted" only.

Comment (suryasr007) [reply _data/projects/RESTEasyCLI.yml:11]: Thanks for the info. Done

**Pull request merged**